website_snippet_anchor/templates/snippets.xml

```diff
@@ -4,13 +4,11 @@
 
 <odoo>
 
-<template id="snippet_options" inherit_id="web_editor.snippet_options">
-    <xpath expr=".">
+<template id="snippet_options" inherit_id="website.snippets">
+    <xpath expr="//div[@id='snippet_options']" position="inside">
         <div data-js="anchor"
              data-selector="section, .carousel">
-            <li data-anchor_ask=""
-                data-only="click"
-                title="Use anchors to go directly to a section of your page, appending #anchor to the URL">
+            <li title="Use anchors to go directly to a section of your page, appending #anchor to the URL">
```

Comment:
I prefer the previous way, since it just follows the standard for snippet options. Please restore it if possible.